DogeNet.DAL/DBContext.cs

```diff
@@ -46,20 +48,26 @@ public DBContext(DbContextOptions<DBContext> options)
 
         public virtual DbSet<Status> Statuses { get; set; }
 
-        public virtual DbSet<User> Users { get; set; }
-
         protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
```

Comment:
Why do we need this method?

Reply:
In case we don't define connection string in Startup

Reply:
But I think it should be deleted